src/main/java/uk/gov/hmcts/reform/notificationservice/data/NotificationRepository.java

```diff
@@ -53,6 +54,14 @@ public NotificationRepository(
         );
     }
 
+    public List<Notification> findByDate(LocalDate date) {
+        return jdbcTemplate.query(
+            "SELECT * FROM notifications WHERE DATE(created_at) = :date",
```

Comment:
we should be excluding description as it may contain sensitive data ?

Reply:
unless it;s going to be mapped to another object dedicated for response. this should be ok

Reply:
I am planning to use existing response which is mapped in controller